test/cets_status_SUITE.erl

```diff
@@ -0,0 +1,388 @@
+-module(cets_status_SUITE).
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("kernel/include/logger.hrl").
+
+-compile([export_all, nowarn_export_all]).
+
+-import(cets_test_setup, [
+    start/2,
+    start_local/1,
+    start_local/2,
+    start_disco/2,
+    make_name/1,
+    make_name/2,
+    disco_name/1
+]).
+
+-import(cets_test_wait, [
+    wait_for_ready/2,
+    wait_till_test_stage/2
+]).
+
+-import(cets_test_setup, [
+    setup_two_nodes_and_discovery/1,
+    setup_two_nodes_and_discovery/2,
+    simulate_disco_restart/1
+]).
+
+-import(cets_test_wait, [
+    wait_for_disco_timestamp_to_appear/3,
+    wait_for_disco_timestamp_to_be_updated/4
+]).
+
+-import(cets_test_receive, [
+    receive_message/1,
+    flush_message/1
+]).
+
+-import(cets_test_peer, [
+    disconnect_node/2,
+    disconnect_node_by_name/2
+]).
```

Comment:
I don't thing these are used. In general, there are some imports in the suites which are not used. It's not a big deal, but is there a reason for it?